### ISSUE TITLE
build-yocto-fsl-arm: Automated build on Docker Hub fails

### ISSUE THREAD

**Issue by gmacario** [gmacario/easy-build#220]
See https://hub.docker.com/r/gmacario/build-yocto-fsl-arm/builds/bnj8l6vvfpwwtfbuwsggdqg/

**Comment by gmacario** [comment]
Fix via 23275b2bf5f37c596638cd75212994e98c9ee55b

**Comment by gmacario** [comment]
Actually the regression was caused by baseimage `gmacario/build-yocto` having changed default USER from `root` to `build`:

```
gmacario@mv-linux-powerhorse:~/easy-build⟫ docker build -t gmacario/build-yocto-fsl-arm build-yocto-fsl-arm/
Sending build context to Docker daemon 47.62 kB
Step 1 : FROM gmacario/build-yocto
 ---> 67fa2d480121
Step 2 : MAINTAINER Gianpaolo Macario <gmacario@gmail.com>
 ---> Using cache
 ---> eb40542c90eb
Step 3 : RUN apt-get -y install curl
 ---> Running in f26a28312023
E: Could not open lock file /var/lib/dpkg/lock - open (13: Permission denied)
E: Unable to lock the administration directory (/var/lib/dpkg/), are you root?
The command '/bin/sh -c apt-get -y install curl' returned a non-zero code: 100
1 gmacario@mv-linux-powerhorse:~/easy-build⟫
```
